src/unix/nto/mod.rs

```diff
@@ -1232,7 +1273,122 @@ pub const MS_SYNC: c_int = 2;
+        pub const TCP_KEEPALIVE: c_int = 0x04;
+        pub const ARPHRD_ARCNET: u16 = 7;
+        pub const SO_BINDTODEVICE: c_int = 0x0800;
+        pub const EAI_NODATA: c_int = 7;
+        pub const IPTOS_ECN_NOT_ECT: u8 = 0x00;
+        pub const RTF_BROADCAST: u32 = 0x80000;
+        pub const UDP_ENCAP: c_int = 100;
+        pub const HW_IOSTATS: c_int = 9;
+        pub const HW_MACHINE_ARCH: c_int = 10;
+        pub const HW_ALIGNBYTES: c_int = 11;
+        pub const HW_CNMAGIC: c_int = 12;
+        pub const HW_PHYSMEM64: c_int = 13;
+        pub const HW_USERMEM64: c_int = 14;
+        pub const HW_IOSTATNAMES: c_int = 15;
+        pub const HW_MAXID: c_int = 15;
+        pub const CTL_UNSPEC: c_int = 0;
+        pub const CTL_QNX: c_int = 9;
+        pub const CTL_PROC: c_int = 10;
+        pub const CTL_VENDOR: c_int = 11;
+        pub const CTL_EMUL: c_int = 12;
+        pub const CTL_SECURITY: c_int = 13;
+        pub const CTL_MAXID: c_int = 14;
+        pub const AF_ARP: c_int = 28;
+        pub const AF_IEEE80211: c_int = 32;
+        pub const AF_NATM: c_int = 27;
+        pub const AF_NS: c_int = 6;
+        pub const BIOCGDLTLIST: c_int = -1072676233;
+        pub const BIOCGETIF: c_int = 1083196011;
+        pub const BIOCGSEESENT: c_int = 1074020984;
+        pub const BIOCGSTATS: c_int = 1082147439;
+        pub const BIOCSDLT: c_int = -2147204490;
+        pub const BIOCSETIF: c_int = -2138029460;
+        pub const BIOCSSEESENT: c_int = -2147204487;
+        pub const FIONSPACE: c_int = 1074030200;
+        pub const FIONWRITE: c_int = 1074030201;
+        pub const IFF_ACCEPTRTADV: c_int = 0x40000000;
+        pub const IFF_IP6FORWARDING: c_int = 0x20000000;
+        pub const IFF_SHIM: c_int = 0x80000000;
+        pub const KERN_ARND: c_int = 81;
+        pub const KERN_IOV_MAX: c_int = 38;
+        pub const KERN_LOGSIGEXIT: c_int = 46;
+        pub const KERN_MAXID: c_int = 83;
+        pub const KERN_PROC_ARGS: c_int = 48;
+        pub const KERN_PROC_ENV: c_int = 3;
+        pub const KERN_PROC_GID: c_int = 7;
+        pub const KERN_PROC_RGID: c_int = 8;
+        pub const LOCAL_CONNWAIT: c_int = 0x0002;
+        pub const LOCAL_CREDS: c_int = 0x0001;
+        pub const LOCAL_PEEREID: c_int = 0x0003;
+        pub const MSG_NOTIFICATION: c_int = 0x0400;
+        pub const NET_RT_IFLIST: c_int = 4;
+        pub const NI_NUMERICSCOPE: c_int = 0x00000040;
+        pub const PF_ARP: c_int = 28;
+        pub const PF_NATM: c_int = 27;
+        pub const pseudo_AF_HDRCMPLT: c_int = 30;
+        pub const SIOCGIFADDR: c_int = -1064277727;
+        pub const SO_FIB: c_int = 0x100a;
+        pub const SO_TXPRIO: c_int = 0x100b;
+        pub const SO_SETFIB: c_int = 0x100a;
+        pub const SO_VLANPRIO: c_int = 0x100c;
+        pub const USER_ATEXIT_MAX: c_int = 21;
+        pub const USER_MAXID: c_int = 22;
+        pub const SO_OVERFLOWED: c_int = 0x1009;
+    } else {
+        pub const SCM_CREDS: c_int = 0x03;
```

Comment:
I randomly sampled a selection of these and they all checked out against my header files.